packages/salesforcedx-vscode-apex-debugger/src/telemetry/telemetry.ts

```diff
@@ -32,9 +31,10 @@ export class TelemetryService {
     this.reporters = reporters;
   }
 
-  public sendExtensionActivationEvent(hrstart: [number, number]): void {
+  public sendExtensionActivationEvent(hrstart?: number): void {
+    const startTime = hrstart || globalThis.performance.now();
```

Comment:
temporarily like this until I find and remove all references to the hrstart param.